modes/color_change_modes.h

```diff
@@ -30,6 +30,10 @@ struct SteppedVariationMode : public SPEC::SteppedMode {
   void next() override { SaberBase::UpdateVariation(1); }
   void prev() override { SaberBase::UpdateVariation(-1); }
 
+  void update() override {
```

Comment:
This all seems good to me, but if we're not going to use say() and fadeout(), then we should probably inherit from SteppedModeBase rather than SteppedMode.

Reply:
oh wait, we don't even need to play ccchange here! I just noticed that makes it play twice because hybrid_font already has this:
```
      case SaberBase::CHANGE_COLOR:
        if (!PlayPolyphonic(&SFX_ccchange)) {
          beeper.Beep(0.05, 2000.0);
        }
```
So no need to call the sound, just inherit from the base instead to avoid both a click and a ccchangeand it's good.

Reply:
:+1: